app/models/card.rb

```diff
@@ -28,7 +33,7 @@ def check_translation(version_of_translation)
   private
 
   def set_default_review_date
-    self.review_date = Time.now + 3.days
+    self.review_date = Time.now.to_date + 3.days
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.